scripts/alert_emails/generate_daily_alerts.ts

```diff
@@ -1,120 +1,101 @@
 /**
```

Comment:
I reworked this script to:
1. Pull the lastSnapshot we sent alerts for from Firestore (see below).
2. Use the summaries JSON files that we're now producing instead of reading the full projections from the API.
3. I dropped changedMetrics from the Alert output out of laziness.